packages/client/hmi-client/src/components/Navbar.vue

```diff
@@ -249,7 +260,7 @@ i {
 /* Suggested terms */
 .suggested-terms {
 	align-items: center;
-	font-weight: var(--font-weight-semibold);
+	font-weight: 400;
```

Comment:
`--font-weight` is 400, and that's the default of the whole app. so you could just remove it.